AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -140,7 +140,13 @@ public class CardComponent: PresentableComponent,
         
         cardViewController.stopLoading()
     }
-    
+
+    public func submit() {
```

Comment:
Is this only available for `CardComponent`? What about other components?

Maybe we should make 2 protocols:
- `SubmitButtonComponent`
- `SubmitButtonComponentConfiguration`

Then make default implementation:

```swift
protocol SubmitButtonComponent: PresentableComponent {
    public func submit()
    public func didSelectSubmitButton()
}

protocol SubmitButtonComponentConfiguration: BasicConfiguration {
    public var showSubmitButton: Bool
}

extension SubmitButtonComponent {
    public func submit() {
         guard 
           let submitConfiguration = (configuration as? SubmitButtonComponentCnfiguration), 
           !submitConfiguration.showSubmitButton
         else { return }
         
         didSelectSubmitButton()
    }
}
```

Reply:
I will take this suggestion in consideration when addressing the submit logic in the next PR